### DISCUSSION TITLE
脚注から本文に戻ったときの遷移位置が分かりづらい

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#818] [要望・アイデア・不具合]
脚注や目次でページ内リンクで遷移したとき、遷移先では遷移先の行が最上部にある状態になります。

脚注から `↩` をクリックして本文に戻るとき、どこに遷移したのか非常に分かりづらいです。スクロールして初めてどこに遷移したのか分かるようになります。
見出しは大きいのでどこに遷移したのか理解しやすいですが、本文中の脚注番号に戻る際はどこに移動したのかわからず戸惑ってしまいます。

どのようにすれば分かりやすいのかは分かりませんが、戻った場所が分かりやすいものになっていると良いなぁと思います。

-----

以下の GIF は脚注に移動してから本文に戻る様子です。<br><br>

![footnote](https://github.com/increments/qiita-discussions/assets/75878492/591b8c82-d095-4ada-9f89-503f9eb104b6 "脚注に移動して本文に戻る様子")

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
ご意見ありがとうございます！
https://github.com/increments/qiita-discussions/discussions/816 と合わせて、何らかの修正を行いたいと思います！🙇

**Reply by kabechiyo** [reply]
@Yarakashi-Kikohshi 
https://github.com/increments/qiita-discussions/discussions/816 の修正により、少しは探しやすくなるのではないかと考えております。
様子をみていただき、不便を感じる場合はまたご意見等お聞かせいただけると幸いです 🙇

**Reply by yarakos95** [reply]
@kabechiyo 
確かに、脚注番号によって角括弧が付くことで以前よりは目立つため探しやすくなったかもしれませんが、私個人としては懐疑的です。あまり変わらないのではとも思っています。

私以外の意見も伺いたいところです。
この手の Discussion がこれまでなかった点を踏まえると、私以外のユーザーは分かりづらいと思っていないかもしれませんが……

-----

私としては、以下のイメージ GIF くらい脚注から本文に戻った時に脚注番号が含まれているパラグラフを強調してほしいと思っています :eyes: 
<br>

![footnote-ideal](https://github.com/increments/qiita-discussions/assets/75878492/969d97f5-2b78-4b1d-a123-4ed7b3f09d33)

**Reply by tanaka-asami415** [reply]
@Yarakashi-Kikohshi
ご確認とGIFの見本もご用意いただき、ありがとうございます！

確かにハイライトだと、とても分かりやすそうです！
他の方のご意見も参考にしつつ、ハイライトの方向で検討したいと思います。

ただ、直近の開発優先度から、すぐの対応ができない可能性があるので、気長にお待ちいただけると幸いです🙇
